snippets/no-decoration-anchor.md

```diff
@@ -0,0 +1,45 @@
+### <a> No Decoration
+
+Removes the default decoration from an `<a>`-tag.
+This is useful when the default highlighting behaviour
+shall not be used.
+
+#### HTML
+
+```html
+<a href="#">
+  I have fancy decorations!
+</a>
+<a class="no-decoration" href="#">
+  I am undecorated!
+</a>
+```
+
+#### CSS
+
+```css
+.no-decoration,
+.no-decoration:hover {
+  color: inherit;
+  text-decoration: none;
```

Comment:
Won't this remove any decoration applied by the surrounding parent, or am I misremembering and this will only remove what the element would apply, and then the parent will decorate it(basically I'm slightly confused on order of operations when it comes to the cascading part of CSS, is it top down or belly up?)

Reply:
Thank you for your feedback! I have double-checked and you are right. I have created [a codepen](https://codepen.io/anon/pen/VEEbeB) to illustrate it.

I checked my recent projects and I have always used this class in combination with some other style (such as Bootstrap's `.font-weight-bold`).

Shall I update the description to make clear it does not take into account surrounding styles and that it just deactivates the default link style?

Reply:
Might be a good idea. The cascade part confuses even the best of us sometimes...

Reply:
That doesn't sound full of myself at all... not one bit... nope nope nope... :humblepie: